src/flows/pipes/Visualization/ErrorThresholds.tsx

```diff
@@ -0,0 +1,378 @@
+// Libraries
+import React, {ChangeEvent, FC, useCallback, useContext, useMemo} from 'react'
+
+// Components
+import {
+  AlignItems,
+  ComponentSize,
+  ComponentStatus,
+  Dropdown,
+  Icon,
+  IconFont,
+  Input,
+  InputType,
+  FlexBox,
+  FlexDirection,
+  Form,
+  TextBlock,
+} from '@influxdata/clockface'
+import {PipeContext} from 'src/flows/context/pipe'
+import {event} from 'src/cloud/utils/reporting'
+
+enum ThresholdFormat {
+  Value = 'value',
+  Range = 'range',
+}
+
+type Threshold = {
+  value: number
+  type: string
+  field: string
+  max?: number
+  min?: number
+}
+
+export const THRESHOLD_TYPES = {
```

Comment:
move this to a shared location to reuse the definitions from the alert panel, or, if you feel that there's isn't enough overlap in the concepts (that threshold means something slightly different here), remove the alert specific information (like condition)